package-res/resources/web/prompting/components/StaticAutocompleteBoxComponent.js

```diff
@@ -92,7 +92,7 @@ define([ 'common-ui/util/util', 'cdf/components/BaseComponent', 'dojo/number', '
               } else {
                 if (Utils.isNumberType(v.type)) {
                   valueParsed = DojoNumber.format(v.label, {
-                    locale : SESSION_LOCALE.toLowerCase()
+                    locale : SESSION_LOCALE.toLowerCase().replace('_', '-')
```

Comment:
I would prefer to have a Session Locale cleanup before.
I always thought dojo handled this properly